src/BenchmarkDotNet/Jobs/JobExtensions.cs

```diff
@@ -330,24 +330,24 @@ public static Job WithEnvironmentVariable(this Job job, [NotNull] string key, [N
         /// <param name="job"></param>
         /// <param name="packageName">The NuGet package name</param>
         /// <param name="packageVersion">The NuGet package version</param>
```

Comment:
The `packageVersion` parameter is also optional now.

```suggestion
        /// <param name="packageVersion">(optional)The NuGet package version</param>
```